gymnasdicts/base.py

```diff
@@ -8,7 +8,7 @@
 JSON = Dict[str, Any]
 
 
-def select(payload: JSON, **pointers: str) -> Iterator[JSON]:
+def select(payloads: Iterator[JSON], **pointers: str) -> Iterator[JSON]:
```

Comment:
this can now only take a "list" of dict in the same manner as `where` and `into`